contract_manager/package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "@certusone/wormhole-sdk": "^0.9.8",
     "@coral-xyz/anchor": "^0.29.0",
-    "@injectivelabs/networks": "1.0.68",
+    "@injectivelabs/networks": "^1.14.6",
```

Comment:
we use this library to get injective grpc endpoints. It's not ideal that it's not defined explicitly and we should later make it explicit.